package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "bedrock-angular-forge",
+  "version": "1.0.0",
```

Comment:
version on implementation branches needs to be 0.0.1-0

The reason for this is that the release system expects the version to be below v1 when we do a 1.0.0 release.

Reply:
wondered about that after seeing a different module, should have asked